README.md

```diff
@@ -1,21 +1,94 @@
+ 
+*Classroom is not an LMS (Learning Management System).* If you need to use an LMS like Canvas, Moodle or Blackboard, we hear you. Students can post their repositories to your LMS. We’re going to stick with what we’re good at, which is helping people collaborate on code. 
+ 
+*Classroom is open source.* Git and GitHub are versatile with many ways to meet your learning goals, and we want to model the open source process that makes our communities great. 
+
+We welcome contributions aligned with the roadmap below and through answering these questions: 
+ 
+* Does it enable the real-life experience of using GitHub as a developer? 
+* Does it replicate functionality of apps, hardware or LMS? 
+* How many support tickets and questions will the feature address? 
+ 
+### Who is Classroom for?
+Anyone teaching computer science in a high school, university or informal environment.
+Folks who might also find Classroom useful:
+* Higher ed statistics and data science teachers
+* Higher ed biology and the hard sciences
+
+## GitHub Classroom and the edtech ecosystem
+
+In case you’re wondering “How does Classroom interact with my favorite app/my notebook/my LMS” here’s the tl;dr on how those pieces fit together: 
+ 
+*Apps and content platforms*
+Examples: Codecademy, Skillshare, Udemy, Udacity
+Apps offer premium content and interactive exercises. GitHub Classroom offers *real-world experience* with code. Classroom, as a teacher-facing application will eventually surface best-in-class content for top courses (notes / lectures / problem sets) but not produce original content. 
+ 
+*Learning Management system/LMS*
+Examples: Blackboard, Moodle, Canvas. Google Classroom
+Teachers often use a learning management system in keeping with student privacy regulations. Classroom has a lightweight integration with LMS ecosystem--students can submit a link to their repositories. LTI compliance and Google Classroom integration will make the app more extensible. 
+ 
+*Notebooks* 
+Examples: BlueJ, Jupyter, RStudio
+Most notebooks have a Git integration that students can push to. Future iterations may pre-populate repos with directions on set up.
+ 
+*Hardware*
+Examples: Chromebooks, Raspberry Pi, Lego
+GitHub Classroom is hardware-agnostic. Shared machines or lab environments are encouraged to use cloud-based environments, like Cloud 9. Integration looks like Git and GitHub pre-loaded + embedded in hardware. 
+ 
+*Assessment*
+Examples: Pearson, Travis CI, Circle CI
+For GitHub Classroom, assessment is **directly related to the real-world experience of being a developer: your code passes tests**. Configuring folders in student repositories is a priority on the roadmap.
+
+## The technical details
+GitHub Classroom is a [Ruby on Rails](http://rubyonrails.org/) application.
 
-### Get started
 New to Ruby? No worries! You can follow these instructions to install a local server.
 
 #### Installing a Local Server
```

Comment:
Putting what @tarebyte said in a comment, it might be nice to extract all the technical stuff into a "How to get started" or "How to contrubute" documents that can live in `docs/` and be linked from here.